internal/controller/rsm/enqueue_ancestor.go

```diff
@@ -17,7 +17,7 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
add `doc.go` to this package, hard to guess `rsm` acronym.

Reply:
done: [3c434dd](https://github.com/apecloud/kubeblocks/pull/3733/commits/3c434ddf15ff9c148fa30cbfe4ddf912c0bd8b23)